packages/network-store/src/network.store.ts

```diff
@@ -78,6 +78,21 @@ export class NetworkStore {
     this.initiateNetworks();
   }
 
+  private getInitialNetworkId() {
```

Comment:
so initial value was default one and then it was reading a different value from localstorage